Makefile

```diff
@@ -6,7 +6,7 @@ PGO_IMAGE_PREFIX ?= crunchydata
 PGO_IMAGE_TAG ?= $(PGO_BASEOS)-$(PGO_VERSION)
 PGO_VERSION ?= $(shell git describe --tags)
 PGO_PG_VERSION ?= 13
-PGO_PG_FULLVERSION ?= 13.4
+PGO_PG_FULLVERSION ?= 13.7
```

Comment:
Can we update this to the latest PG 14 version instead?

Reply:
will do

Reply:
Updated